src/test/java/jimmy/mcgymmy/model/food/ProteinTest.java

```diff
@@ -13,9 +13,9 @@ public void constructor_null_throwsNullPointerException() {
     }
 
     @Test
-    public void constructor_invalidProtein_throwsIllegalArgumentException() {
+    public void constructor_invalidProtein_throwsNumberFormatException() {
         String invalidProtein = "";
-        assertThrows(IllegalArgumentException.class, () -> new Protein(invalidProtein));
+        assertThrows(NumberFormatException.class, () -> new Protein(invalidProtein));
```

Comment:
Leaving this in for now, but we should depreciate this constructor in another PR. see #214